tests/configuration_endpoint.tests/multiple.json

```diff
@@ -0,0 +1,15 @@
+{
+  "protocol": "mqtt",
+  "hostname": "mqtt.googleapis.com",
+  "port": 8883,
+  "client_id": "prasjdhsadas",
```

Comment:
What is this - just a placeholder?

Reply:
The file is just for testing against various kinds of schema violations.... not even sure that it counts as a placeholder.